src/mpyl/cli/commands/projects/lint.py

```diff
@@ -142,7 +144,7 @@ def _lint_whitelisting_rules(
     projects: list[Project],
     config: dict,
     target: Target,
-):
+) -> bool:
```

Comment:
This feels a bit hacky, also considering the function name and the negative return. I think returning the list and letting the calling functionality handle the printing as well would fix this (for both functions).